test/config/chaosduck.yaml

```diff
@@ -81,8 +81,6 @@ spec:
         # Disable chaos on the Istio components.
         - "-disable=istio-webhook"
         - "-disable=networking-istio"
-        # Disable chaos on Kourier components.
-        - "-disable=3scale-kourier-control"
```

Comment:
Do we also need to add it to the HA_COMPONENTS variable, to make sure stuff gets scaled accordingly?

Reply:
Yes! Thank you. I missed it.